static/js/image_upload_widget.js

```diff
@@ -15,7 +15,9 @@ exports.set_up = function () {
         const form_data = new FormData();
         let url;
         let night_logo = false;
-        if (widget === "realm-icon-upload-widget") {
+        if (widget === "user-avatar-upload-widget") {
```

Comment:
Hacky. Let's make `ImageUploadWidget` a class and then have 2 (or more) instances of it that each take in a `section` parameter.

Let's say your final export could look like:

```
exports.realm_icon = ImageUploadWidget('realm-icon');
exports.user_avatar = ImageUploadWidget('user-avatar');
```

Then any uses of this could be like:

```
image_upload_widget.realm_icon.open();
```

Another similar strategy is to declare these widgets in the files that need them. In that case, your export would look like:

```
exports = ImageUploadWidget
```

And your uses would look like:

```
const realm_icon_upload_widget = image_upload_widget('realm-icon');
realm_icon_upload_widget.open();
```